### PR TITLE
Various changes in prep for opcode & conditions renumbering.

### PR DESCRIPTION
**Pull request by richardkiss**

### CODE REVIEW

**Comment by richardkiss** [comment]
If tests all pass, this is ready for rebase merge. This is just prep for renumber, it doesn't yet do the renumber. If tests fail, I'll rework until they pass.

File: src/wallet/puzzles/recompile-all.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# This hack is a quick way to recompile everything in this directory
+
+#BASE_DIR=`pwd | dirname`
```
Comment (hoffmang9) [inline src/wallet/puzzles/recompile-all.sh:5]: per my message there are a couple of errors here that ShellCheck is going to raise in lint. https://www.shellcheck.net/

**Review by hoffmang9 — APPROVED** [review]
Other than the shellcheck lint issue - looks good to me

File: src/wallet/puzzles/recompile-all.sh (hunk 1 above)
Comment (richardkiss) [reply src/wallet/puzzles/recompile-all.sh:5]: Thanks for that.

**Pull request merged**